.github/workflows/publish-rustdocs.yml

```diff
@@ -42,13 +53,50 @@ jobs:
           docker run --rm \
             -v "${GITHUB_WORKSPACE}:/workspace" \
             -v "${HOME}/.cargo:/root/.cargo" \
-            -w /workspace \
             -e RUSTDOCFLAGS='-D warnings' \
+            -w /workspace \
             paritytech/ci-unified:bullseye-1.74.0 \
-            bash -c "cargo doc --all-features --no-deps --locked && chown -R $(id -u):$(id -g) ${{ env.DOCS_TARGET }}"
+            bash -c "
+              cargo doc --all-features --no-deps --locked && \
+              mv $DOCS_TARGET/help.html $DOCS_TARGET/index.html && chown -R $(id -u):$(id -g) $DOCS_TARGET"
```

Comment:
Just move the third command on its own line as well.

Reply:
Done